advanced/internal-image-registry.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Internal Image Registry
+---
+
+# Kontena Internal Image Registry
+
+Kontena has built-in support for creating an Internal Image Registry. The Internal Image Registry may be used to store Docker images used by your own application. Kontena's Internal Image Registry runs on the same infrastructure and private network as your Nodes. Therefore, access to Kontena's Image Registry is not open publicly and may be accessed only by using [VPN access](../tools/vpn-access.md). This isolation ensures that you will have total control over the access control, security and distribution of your images. Kontena Image Registry is based on [Docker Image Registry](https://docs.docker.com/registry/).
+
+Kontena may be used with any Docker Image Registry. Users looking for a zero-maintenance, ready-to-go solution are encouraged to consider [Kontena Cloud](/tools/image-registry.md), [Docker Hub](https://hub.docker.com/account/signup/) or [Quay](https://quay.io/). Both of these services provide a hosted registry, as well as some advanced features.
+
+You should use Kontena's built-in Image Registry if you want to:
+
+* Have total control over where your images are being stored
+* Fully own your images' distribution pipeline
+* Ensure access control and security for your own Docker images
+
```

Comment:
> Fully own your images' distribution pipeline
and
> Ensure access control and security for your own Docker images

Can't one accomplish both using KCloud Registry too? I kinda get the point, but these statements might make someone wonder if KCloud registry does not provide access control and security. :)